wsmaster/che-core-api-workspace/src/main/java/org/eclipse/che/api/workspace/devfile/server/Constants.java

```diff
@@ -9,7 +9,7 @@
  * Contributors:
  *   Red Hat, Inc. - initial API and implementation
  */
-package org.eclipse.che.api.devfile.server;
+package org.eclipse.che.api.workspace.devfile.server;
```

Comment:
Shared classes are in `org.eclipse.che.api.workspace.shared.dto.devfile` package.

Stack related classes are is `org.eclipse.che.api.workspace.server.stack` package.

Maybe it would be better to name devfile server package as ` org.eclipse.che.api.workspace.server.devfile`?

Reply:
Also, devfile related impls are in `org.eclipse.che.api.workspace.server.model.impl.devfile`

Reply:
Since Kubernetes component related classes are moved to k8s-infra module, I think it makes sense to move related constants as well.
It's not a requirement but just a recommendation to consider doing it since I realize that it may require a lot of additional work that is not so needed.

Reply:
> Since Kubernetes component related classes are moved to k8s-infra module, I think it makes sense to move related constants as well.
It's not a requirement but just a recommendation to consider doing it since I realize that it may require a lot of additional work that is not so needed.

@metlos Could you please leave your opinion about it? The comment is resolved but plan about this topic is not clear.